Makefile

```diff
@@ -0,0 +1,123 @@
+DC_CI = "bin/dc.sh"
+DC = $(shell which docker-compose)
+
+default: help
+	@echo ""
+	@echo "You need to specify a subcommand."
+	@exit 1
+
+help:
+	@echo "build         - build docker images for dev"
+	@echo "run           - docker-compose up the entire system for dev"
+	@echo ""
+	@echo "pull          - pull the latest production images from Docker Hub"
+	@echo "init          - initialize the database and install Node and Bower packages"
+	@echo "djshell       - start a Django Python shell (ipython)"
+	@echo "shell         - start a bash shell"
+	@echo "clean         - remove all build, test, coverage and Python artifacts"
+	@echo "rebuild       - force a rebuild of all of the docker images"
+	@echo "lint          - check style with flake8, jshint, and stylelint"
+	@echo "test          - run tests against local files"
```

Comment:
I get a bunch of errors about missing static files running this, but that may not be related to this PR. (Local testing has never been solid for me.)

Reply:
Also failed with `MySQL server has gone away`, but I'm pretty sure that's not related to this PR.

Reply:
hmmm... maybe we do need a collectstatic step? And I have been trying to figure out the "MySQL server has gone away" thing, but it's still not fixed :( My suspicion right now is some strange interaction with django-nose. It is not a quick and easy thing to move to pytest for a test though.